source/index.html.md

```diff
@@ -1,239 +1,504 @@
-```
+## <span class="get">GET</span> Generic Endpoint
 
-```python
-import kittn
+>https://api.namara.io/v0/:endpoint
 
-api = kittn.authorize('meowmeowmeow')
-```
+Generic endpoint. Chart displays standard response codes:
 
-```shell
-# With shell, you can just pass the correct header with each request
-curl "api_endpoint_here"
-  -H "Authorization: meowmeowmeow"
-```
+Request | Response | Description
+------- | -------- | -----------
+ | 200: OK | Success
+ | 202: Accepted | A background job has successfully been triggered. Continue polling for details.
+ | 401: Unauthorized | The API key you provided doesn't correspond to any user.
+ | 403: Forbidden | User is not authorized to make that request.
+ | 422: Unprocessable Entity | The server was unable to save the document. There will be more details in the full response body.
 
-```javascript
-const kittn = require('kittn');
 
-let api = kittn.authorize('meowmeowmeow');
-```
+## API Keys
 
-> Make sure to replace `meowmeowmeow` with your API key.
+Three kinds of API Keys can be created for accessing the Namara API - Organization Keys, Project Keys, and Personal Keys. This is an overview of managing all three types, and we'll outline the different use cases for each.
 
-Kittn uses API keys to allow access to the API. You can register a new Kittn API key at our [developer portal](http://example.com/developers).
+### Organization API Keys
 
-Kittn expects for the API key to be included in all API requests to the server in a header that looks like the following:
+For any non-personal Organization, visit the "Settings" tab and create API Keys for that Organization. Calls made with this API Key will be limited to Data Sets and Projects that your Organization has access to.
 
-`Authorization: meowmeowmeow`
+### Project API Keys
 
-<aside class="notice">
-You must replace <code>meowmeowmeow</code> with your personal API key.
-</aside>
+For any project, visit the "Settings" tab and create API Keys for yourself for that Project. Calls made with this API Key will be limited to Data Sets that your Project has access to.
 
-# Kittens
+### Personal API Keys
 
-## Get All Kittens
+The personal Namara API Key can be obtained by clicking on the Organization tab at the bottom of the left sidebar, then clicking "Account Settings". Copy this from the "API Key" tab and use it for any API calls.
 
-```ruby
-require 'kittn'
+<aside class="warning">This key gives API access to the entire Namara account. If you are planning on giving your API Key to another user, we would recommend providing them with an Organization API Key or a Project API Key in order to limit their access to your account.</aside>
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get
-```
+### Rate Limiting
 
-```python
-import kittn
+Users are limited to 10,000 requests per month, as well as 100 data set downloads per month. If you exceed this limit, the API will return `status: 429`
```

Comment:
Let's add this to the status table above too

Reply:
Done